dspace-api/src/main/java/org/dspace/content/service/CommunityService.java

```diff
@@ -292,4 +293,13 @@ public void removeSubcommunity(Context context, Community parentCommunity, Commu
     public List<Community> findAuthorizedGroupMapped(Context context, List<Integer> actions) throws SQLException;
 
     int countTotal(Context context) throws SQLException;
+
+    /**
+     * Returns total community archived items
+     *
+     * @param community       Community
+     * @return                total community archived items
+     * @throws ItemCountException
+     */
+    int getArchivedItems(Community community) throws ItemCountException;
```

Comment:
I'd recommend renaming this `countArchivedItems()` so that it's the same name as in `CollectionService`.  Methods that do the same thing should have the same name.